simple_learn/regressors/simple_regressor.py

```diff
@@ -128,9 +128,9 @@ def fit(self, train_x, train_y, folds=3):
                 start = time.time()
                 try:
                     grid_clf.fit(train_x, train_y)
-                except ValueError as value_error:
+                except BaseException as error:
                     self.failed_models.append(name)
-                    self.logger.warning(f"{name} failed, Error : {value_error}")
+                    self.logger.warning(f"{name} failed due to, Error : {error} and another model was fitted.")
```

Comment:
You can go ahead and remove "and another model was fitted." There might be multiple model algorithms that fail.